gradle/verification-metadata.xml

```diff
@@ -0,0 +1,3608 @@
+      <verify-signatures>false</verify-signatures>
+      <trusted-artifacts>
+         <trust group="org.elasticsearch.ml"/>
+         <trust group="org.elasticsearch.plugin"/>
+         <trust group="org.elasticsearch"/>
+         <trust group="org.elasticsearch.distribution.zip" name="elasticsearch"/>
+         <trust group="elasticsearch-distribution" name="elasticsearch"/>
+         <trust file=".*-javadoc[.]jar" regex="true"/>
+         <trust file=".*-sources[.]jar" regex="true"/>
+      </trusted-artifacts>
+   </configuration>
+   <components>
+      <component group="adoptium_16" name="linux" version="16.0.2">
+         <artifact name="linux-16.0.2-x64.tar.gz">
+            <sha256 value="323d6d7474a359a28eff7ddd0df8e65bd61554a8ed12ef42fd9365349e573c2c" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="adoptium_16" name="windows" version="16.0.2">
+         <artifact name="windows-16.0.2-x64.zip">
+            <sha256 value="40191ffbafd8a6f9559352d8de31e8d22a56822fb41bbcf45f34e3fd3afa5f9e" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="adoptium_17" name="linux" version="17.0.2">
+         <artifact name="linux-17.0.2-x64.tar.gz">
+            <sha256 value="288f34e3ba8a4838605636485d0365ce23e57d5f2f68997ac4c2e4c01967cd48" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="adoptium_17" name="windows" version="17.0.2">
+         <artifact name="windows-17.0.2-x64.zip">
+            <sha256 value="d083479ca927dce2f586f779373d895e8bf668c632505740279390384edf03fa" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="adoptium_8" name="linux" version="8u302">
+         <artifact name="linux-8u302-x64.tar.gz">
+            <sha256 value="cc13f274becf9dd5517b6be583632819dfd4dd81e524b5c1b4f406bdaf0e063a" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="adoptium_8" name="mac" version="8u302">
+         <artifact name="mac-8u302-x64.tar.gz">
+            <sha256 value="6205fefca28342d99938b3933ed839784835ed1de6ed9ba034ce772377f74061" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="aopalliance" name="aopalliance" version="1.0">
+         <artifact name="aopalliance-1.0.jar">
+            <sha256 value="0addec670fedcd3f113c5c8091d783280d23f75e3acb841b61a9cdb079376a08" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="asm" name="asm" version="3.1">
+         <artifact name="asm-3.1.jar">
+            <sha256 value="333ff5369043975b7e031b8b27206937441854738e038c1f47f98d072a20437a" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="beats" name="filebeat" version="8.4.0-SNAPSHOT">
+         <artifact name="filebeat-8.4.0-SNAPSHOT-x86_64.tar.gz">
```

Comment:
For release builds, we'll depend on the non-snapshot version, obviously. Also since this is a snapshot, the checksum will change all the time. Same for `metricbeat`.

Reply:
Should we even have `filebeat` and `metricbeat` in here? Can we skip some dependencies? Also we can't know the correct SHAs for the release versions until the release has happened, which is too late if the sums are checked during the build.

Reply:
@pugnascotia fixed